deployment/base/hg-deployment.yaml

```diff
@@ -149,6 +149,14 @@ spec:
         ports:
           - containerPort: 80
 
+        livenessProbe:
+          httpGet:
+            path: /api/v03/isAvailable
+            port: 80
+          failureThreshold: 5
+          periodSeconds: 120
```

Comment:
These values seem kind of high to me 🤔. It could be down for up to 12 minutes before it gets restarted.

Reply:
Bikeshedding welcome, I just figured we don't want to hit the endpoint very often and we want to allow 10 minutes before we decide the container is stuck. If those numbers should be a lot shorter it's a trivial change to make. Any thoughts, @hahn-kev?

Reply:
I'm fine with it as is. We can tweak it once we have more information, for now we're just shooting in the dark on the time.